modules/javafx.swing/src/main/java/javafx/embed/swing/JFXPanel.java

```diff
@@ -135,6 +135,14 @@
  *         }
  *     }
  * </pre>
+ * <strong>Warning:</strong>
+ * Serialized objects of this class will not be compatible with
+ * future Swing releases. The current serialization support is
+ * appropriate for short term storage or RMI between applications running
+ * the same version of Swing.  As of 1.4, support for long term storage
```

Comment:
Add "and the same version of JavaFX" after Swing.

I would remove everything after the "As of 1.4 ..." since that isn't really relevant for JavaFX components and could be confusing.